GitVersion.yml

```
@@ -0,0 +1,35 @@
+# current development version which is first version of next release branched away from master
+next-version: 1.11.0
+
+# configuration for all branches
+assembly-versioning-scheme: MajorMinorPatchTag
+assembly-file-versioning-scheme: MajorMinorPatchTag
+mode: ContinuousDeployment
+legacy-semver-padding: 5
+build-metadata-padding: 5
+commits-since-version-source-padding: 5
+commit-message-incrementing: Disabled
+
+branches:
+  master:
+    regex: master
```

```suggestion
    regex: ^master$
```
This ensures that branch names just _containing_ the string "master" do not match this pattern.